WalletWasabi.Tests/UnitTests/Microservices/ProcessBridgeTest.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Microservices;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.Microservices
+{
+	public class ProcessBridgeTest
+	{
+		[Fact]
+		public async void SendCommandSuccessAsync()
+		{
+			using var cts = new CancellationTokenSource();
+
+			var processBridge = new ProcessBridge(MicroserviceHelpers.GetBinaryPath("hwi"));
+			(string response, int exitCode) p = await processBridge.SendCommandAsync("--help", openConsole: false, cts.Token);
+
+			Assert.Equal(0, p.exitCode);
+			Assert.Equal("{\"error\": \"Help text requested\", \"code\": -17}" + Environment.NewLine, p.response);
+		}
+
+		[Fact]
+		public async void SendCommandImmediateCancelAsync()
+		{
+			await Assert.ThrowsAsync<TaskCanceledException>(async () =>
+			{
+				using var cts = new CancellationTokenSource();
+				cts.Cancel();
+				var processBridge = new ProcessBridge(MicroserviceHelpers.GetBinaryPath("bitcoind"));
+				(string response, int exitCode) p = await processBridge.SendCommandAsync("-testnet", openConsole: false, cts.Token);
```

Comment:
Is this supposed to be `--testnet` or `-testnet`?

Reply:
--testnet